apps/nrm/fields/index.js

```diff
@@ -9,10 +9,10 @@ const dateComponent = require('hof').components.date;
 const ukLocalAuthorities = require('../../../ms-lists/ms_uk_local_authorities');
 const msNationalities = require('../../../ms-lists/ms_nationalities');
 const organisations = require('../../../ms-lists/ms_organisations');
-const path = require('path');
 
 module.exports = {
   reference: {
+    className: ['govuk-input', 'govuk-!-width-one-half'],
```

Comment:
Is the `'govuk-!-width-one-half'` class needed?

Reply:
Discussed this previously. Set the width here to match the width in UAT. Removed the width style class from the other fields above.